working_process.md

```diff
@@ -0,0 +1,40 @@
+# Working Process
+
+These guidelines applied to **all** CocoaHeads Russia projects.
+
+## Xcode and Swift Style Guidelines
+
+**We follow [these guidelines](https://github.com/raywenderlich/swift-style-guide) in coding**
+
+* Don't use acronyms and abbreviations
+* Use indentation == 2 in for *Tab width* and *Indent width*
+
+**Xcode → Preferences → Text Editing → Indentation**
```

Comment:
We can mention that it should be turned on to project automatically, so there shouldn’t be any problem. Unfortunately VAPOR generates Xcode project from scratch each time with values from Xcode settings.

Reply:
We can change this configuration with our shell script. We'll write it later.

Reply:
Fixed